exec.go

```diff
@@ -19,6 +19,15 @@ import (
 var execCommand = cli.Command{
 	Name:  "exec",
 	Usage: "execute new process inside the container",
+	ArgsUsage: `<container-id> <container command>
+
+Where "<container-id>" is the name for the instance of the container and
+"<container command>" is the command to be executed in the container.
+
+For example, if the container is a linux distro the following will output a list
```

Comment:
I would reword this as the container doesn't necessarily have to be a distro. Even a busybox rootfs has the ps command.